src/serac/physics/solid_mechanics.hpp

```diff
@@ -818,7 +819,8 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
   {
     residual_->AddDomainIntegral(
         Dimension<dim>{}, DependsOn<0, 1, 2, active_parameters + NUM_STATE_VARS...>{},
-        [body_force, this](auto x, auto /* displacement */, auto /* acceleration */, auto shape, auto... params) {
+        [body_force, this](double /*t*/, auto x, auto /* displacement */, auto /* acceleration */, auto shape,
```

Comment:
We should remove the `this` capture to ensure we aren't aliasing host memory.

Reply:
whoops, missed that one!